### PR TITLE
jami: 20220825.0828.c10f01f -> 20221031.1308.130cc26

### PR DESCRIPTION
**Pull request by linsui**
###### Description of changes
jami-client-qt is renamed to jami-client. https://git.jami.net/savoirfairelinux/jami-client-qt/-/commit/bf4a8c314f308877841dcfa8c567cdd097c0fbb0
withWebengine option is removed due to an upstream bug. https://git.jami.net/savoirfairelinux/jami-client-qt/-/issues/881

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by foo-dogsquared — APPROVED** [review]

File: pkgs/applications/networking/instant-messengers/jami/client.nix [hunk 1]
```diff
@@ -21,17 +21,17 @@
 , qtsvg
 , qtwebengine
 , qtwebchannel
-, withWebengine ? true
+# https://git.jami.net/savoirfairelinux/jami-client-qt/-/issues/881
+# , withWebengine ? false
```
Comment (SuperSandro2000) [inline pkgs/applications/networking/instant-messengers/jami/client.nix:25]: https://git.jami.net/savoirfairelinux/jami-client-qt/-/commit/9b2dbb64eaa9256f800dfa69d897545f4b0affd2

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: pkgs/applications/networking/instant-messengers/jami/client.nix (hunk 1 above)
Comment (linsui) [reply pkgs/applications/networking/instant-messengers/jami/client.nix]: Thanks! Patch added.

**Comment by linsui** [comment]
@SuperSandro2000 Is this good to merge?

**Comment by SuperSandro2000** [comment]
@ofborg build jami-daemon jami-client

**Comment by linsui** [comment]
Thanks!

**Pull request merged**